spec/requests/api/v1/moves_controller_role_access_spec.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Api::V1::MovesController, with_client_authentication: true do
+  let(:headers) { { 'CONTENT_TYPE': content_type }.merge(auth_headers) }
+  let(:content_type) { ApiController::CONTENT_TYPE }
+  let(:response_json) { JSON.parse(response.body) }
+
+  describe 'GET /moves' do
+    let!(:application) { Doorkeeper::Application.create(name: 'test', owner: supplier) }
+
+    let(:schema) { load_json_schema('get_moves_responses.json') }
+
+    let(:supplier) { create :supplier }
+    let(:pentonville) { create :location, suppliers: [supplier] }
+    let(:birmingham) { create :location, key: 'hmp_birmingham', title: 'HMP Birmingham', nomis_agency_id: 'BMI' }
+    let!(:pentonville_moves) { create_list :move, 10, from_location: pentonville }
+    let!(:birmingham_moves) { create_list :move, 10, from_location: birmingham }
```

Comment:
Could this be a `with_moves` trait on the `location` factory? It feels like we do/should need suppliers with moves for a bunch of our tests.

Reply:
Done in the supplier filter branch, will "integrate" properly once the other PR reaches master.

Reply:
:+1: